utils/auth.py

```
@@ -0,0 +1,76 @@
+
+    Args:
+        data: 토큰에 담을 데이터
+        expires_delta: 만료 시간
+
+    Returns:
+        JWT 토큰 문자열
+    """
+    to_encode = data.copy()
+
+    if expires_delta:
+        expire = datetime.utcnow() + expires_delta
```

Ruff (0.14.13)

`datetime.datetime.utcnow()` used

(DTZ003)